x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.test.tsx

```diff
@@ -33,7 +33,7 @@ describe('EmptyEngineOverview', () => {
 
   it('renders a documentation link', () => {
     expect(getPageHeaderActions(wrapper).find(EuiButton).prop('href')).toEqual(
-      `${docLinks.appSearchBase}/index.html`
+      `${docLinks.appSearchGuide}`
```

Comment:
```suggestion
      docLinks.appSearchGuide
```
No need for a template literal here